### PR TITLE
feat: add learning rate scheduler in TF training scripts

### PR DESCRIPTION
**Pull request by charlesmindee**
This PR adds a CosineDecay lr scheduler in both tensorflow training scripts.
Any feedback is welcome!

### CODE REVIEW

**Review by fg-mindee — APPROVED** [review]
Nice, thanks for this improvement!
